### PR TITLE
Omit latest Pandas version

### PR DESCRIPTION
**Pull request by jeremyliweishih**
Omit latest Pandas ver

### CODE REVIEW

**Review by dsherry — APPROVED** [review]
LGTM 🚢

**Review by dsherry — APPROVED** [review]
👍  looks good!

**Comment by jeremyliweishih** [comment]
@dsherry think I need to specify graphviz version for the circleci tests as well. It's pulling 0.16 as we don't specify any version rn.

https://app.circleci.com/pipelines/github/alteryx/evalml/8385/workflows/7fd82183-536b-458a-b130-cb0696beeae3/jobs/96840/steps

**Comment by dsherry** [comment]
@jeremyliweishih yep you beat me to it :) https://github.com/alteryx/evalml/blob/main/.circleci/config.yml#L154

File: .circleci/config.yml [hunk 1]
```diff
@@ -155,7 +155,7 @@ jobs:
           command: |
               C:\Users\circleci\Miniconda3\shell\condabin\conda-hook.ps1
               conda activate curr_py
-              conda install python-graphviz -q -y
+              conda install python-graphviz=0.15 -q -y
```
Comment (dsherry) [inline .circleci/config.yml:158]: 👍

**Pull request merged**